BetterMD/elements/document.py

```
@@ -63,9 +31,12 @@ def copy(self):
 HASHABLE_ATTRS = str | bool | int | float | HashableList['HASHABLE_ATTRS'] | HashableDict[str, 'HASHABLE_ATTRS']
```

Ruff (0.8.2)

Undefined name `HashableList`

(F821)